nnpdf_data/nnpdf_data/athena_utils.py

```diff
@@ -0,0 +1,131 @@
+import pandas as pd
+from pathlib import Path
+from typing import Optional, Union
+
+import yaml
```

Comment:
```suggestion
from reportengine.compat import yaml
```

Reply:
Ah no, reportengine is not a dependency and perhaps the standard library is good enough here